### DISCUSSION TITLE
Desktop Environments

### DISCUSSION THREAD

**Discussion by TheDreadStorm** [linuxmint/discussions#481] [Questions]
I'm brand new to Linux, but I already know to stay away from certain builds, so I settled on LMDE. Of all the distros I've tested, I decided I prefer the XFCE desktop. I absolutely despise anything that too closely resembles a Mac, and KDE is on top of that list. Not too fond of Cinnamon, either, really, but it'll do if I have no choice. I haven't installed LMDE yet, but I'm in the process of reading up on it.

So I have a question, in multipart. Only one needs answered.
Are there any plans to make an XFCE build of LMDE, or is it an option during install?
- or - 
Can it be installed reasonably automatically? And if so, where can I go to obtain instructions on doing so?

Thank you for your attention in this. :)  Like I said, I'm brand-new, so I barely have a clue of what the hell I'm doing. heh

**Comment by rcalixte** [comment]
> * Can it be installed reasonably automatically? And if so, where can I go to obtain instructions on doing so?

You can install [task-xfce-desktop](https://packages.debian.org/bookworm/task-xfce-desktop) which will pull in all of the dependencies for XFCE. Once the installation is completed, you would log out and log back in again, making sure to select the XFCE desktop. You would want to keep Cinnamon installed to avoid any potential issues.

Rather than doing all of that though, if you want a Debian base and XFCE, you can [install Debian directly and select XFCE](https://wiki.debian.org/Xfce#Install_a_new_Debian_system_with_Xfce) from the installer options. The installation and the default applications will slightly differ from LMDE but unless you need/want something specific from Mint, you would be able to install/uninstall mostly the same software.



> I'm brand-new, so I barely have a clue of what the hell I'm doing. heh

Welcome and good luck on your journey! Don't expect to have everything be perfect the first time around. Be agile, willing to learn, willing to adjust, and willing to experiment and you'll be fine in the long run.
